server/routerlicious/packages/services-shared/src/socketIoServer.ts

```diff
@@ -321,8 +321,12 @@ class SocketIoServer implements core.IWebSocketServer {
 			}
 		}
 
-		this.io.close();
-		await sleep(3000); // Give time for any  disconnect handlers to execute before closing Redis resources
+		await this.io.close();
```

Comment:
Necessary due to breaking change in socket.io `4.7.5->4.8.0`.

Reply:
@znewton any thoughts on this change?

Reply:
I feel decently confident that this shouldn't have a big impact so I'll go ahead merging.